server/perkeepd/perkeepd.go

```diff
@@ -96,9 +96,9 @@ var (
 	flagHelp       = flag.Bool("help", false, "show usage")
 	flagLegal      = flag.Bool("legal", false, "show licenses")
 	flagConfigFile = flag.String("configfile", "",
-		"Config file to use, relative to the Perkeep configuration directory root. "+
-			"If blank, the default is used or auto-generated. "+
-			"If it starts with 'http:' or 'https:', it is fetched from the network.")
+		"Config file to use, relative to the Perkeep configuration directory root specified by this environnement variable CAMLI_CONFIG_DIR defled to $HOME/.config/perkeep. "+
```

Comment:
Please use the PerkeepConfigDir function,

`Config file to use, relative to the Perkeep configuration directory root specified by the CAMLI_CONFIG_DIR environment varaible (currently <PerkeepConfigDir>)`